### DISCUSSION TITLE
Trascr_Matr_999: indicazioni sulla valorizzazione di "idTipoContenuto"

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#760] [Supporto servizi cooperativi] [answered]
Nell'utilizzo del caso d'uso "Trascr_Matr_999", sorge un dubbio circa la valorizzazione del campo
"idTipoContenuto"
riferito a decodifica:

![image](https://github.com/italia/ansc/assets/26415430/36f35375-dae9-4173-9601-74db4d33801e)

Facciamo, infatti, riferimento a "Trascr_Matr_999" per situazioni diverse; in particolare
a) per atti che il Comune in ANSC rediga per trascrivere atto di stato civile che altro Comune italiano abbia prodotto in originale (cartaceo). Iin questo caso pare corretto che "idTipoContenuto" assuma valore "3" e, procedendo effettivamente secondo questa valorizzazione, non si riscontra alcun'anomalia.
b) atti che il Comune in ANSC formi a seguito di richiesta da parte del soggetto privato che ha interesse alla trascrizione; si pensi, ad esempio, al caso di cittadino che acquista la cittadinanza italiana e chiede la trascrizione dall'estero del proprio atto di matrimonio.
Anche in questo caso il servizio si aspetta "idTipoContenuto" assuma valore "3", ma abbiamo il dubbio che sarebbe in effetti corretto il valore "1": l'atto che il Comune redige è frutto di un'operazione di "trascrizione" ma è il solo atto (originale quindi) di stato civile presente in Italia
c) atti che il Comune in ANSC formi a seguito di richiesta da parte di autorità terza; si pensi, ad esempio, al caso di sentenza di divorzio emessa da Tribunale italiano.
Anche in questo caso il servizio si aspetta "idTipoContenuto" assuma valore "3", ma abbiamo il dubbio che sarebbe in effetti corretto il valore "1": l'atto che il Comune redige è frutto di un'operazione di "trascrizione" ma è il solo atto (originale quindi) di stato civile presente in Italia per quel tipo di procedimento.

Diversamente, per Tascr_Matr_999, in quali casi è previsto che "idTipoContenuto" assuma valore "1"?

**Comment by amusarra** [comment, marked as answer]
Salve @antonioluberto 
Così come indicato dalla tabella di decodifica, per tutti gli US di trascrizione deve essere usato per `idTipoContenuto` il valore indicato, ovvero, `3`.

Cordiali Saluti.
Supporto ANSC.
